pkg/collector/corechecks/ebpf/c/runtime/oom-kill-kern.c

```diff
@@ -37,39 +38,44 @@ int BPF_KPROBE(kprobe__oom_kill_process, struct oom_control *oc) {
         return 0;
     }
 
-    s->pid = pid;
-    get_cgroup_name(s->cgroup_name, sizeof(s->cgroup_name));
+    // for kernel before 4.11 the prototype for bpf_probe_read helpers
```

Comment:
OOM Kill requires 4.11 https://docs.datadoghq.com/integrations/oom_kill/

Reply:
The code tests for v4.9
@L3n41c should we bump the version in the code?

Reply:
The documentation might not have been updated each time the `oom_kill` check has been refactored.
If we have tests validating that the `oom_kill` check works on `4.9`, which is the version that the implementation checks: https://github.com/DataDog/datadog-agent/blob/f181fade8a1342b39062b4876ed0a4806b4dbdf0/pkg/collector/corechecks/ebpf/probe/oomkill/oom_kill.go#L70, then I think it makes sense to update the doc: DataDog/integrations-core#17639.